src/main/java/net/dv8tion/jda/api/events/self/SelfUpdateAvatarEvent.java

```diff
@@ -60,6 +61,22 @@ public String getOldAvatarUrl()
         return previous == null ? null : String.format(AVATAR_URL, getSelfUser().getId(), previous, previous.startsWith("a_") ? ".gif" : ".png");
     }
 
+    /**
+     * Returns an {@link ImageProxy} for this bot's new avatar image
+     * <p>
+     * <b>Note:</b> the old avatar may not always be downloadable as it might have been removed from Discord
```

Comment:
```suggestion
     * Returns an {@link ImageProxy} for this bot's new avatar image.
     * <p>
     * <b>Note:</b> the old avatar may not always be downloadable as it might have been removed from Discord.
```